pkg/exporter/sumologicexporter/config.go

```diff
@@ -255,4 +260,6 @@ const (
 	DefaultTimestampKey string = "timestamp"
 	// DefaultFlattenBody defines default FlattenBody value
 	DefaultFlattenBody bool = false
+	// DefaultDropRoutingAttribute defines default DropRoutingAttribute
+	DefaultDropRoutingAttribute string = "sumo_routing"
```

Comment:
I wonder if we shouldn't set the default to empty string and don't drop anything then (make it conditional).

@SumoLogic/open-source-collection-team  WDYT?

Reply:
I agree, especially if we do not target it by defaut to `http_listener_v2_path`

Reply:
fixed